ui/raidboss/data/06-ew/raid/p3n.ts

```diff
@@ -14,21 +14,13 @@ const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.AsphodelosTheThirdCircle,
   triggers: [
     {
-      id: 'P3N Experimental Fireplume 1',
+      id: 'P3N Experimental Fireplume Out',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ id: '6698', source: 'Phoinix', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ id: '6698', source: 'Protophénix', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ id: '6698', source: 'フェネクス', capture: false }),
-      response: Responses.getInThenOut(),
```

Comment:
I think "in then out" is technically true, but is not a raid call that made much sense to me.  It's really more like "get middle and then run out to where the third explosion was while dodging the other outside explosions".

I think this is just a "dodge visible telegraphs that don't need a trigger" sort of mechanic, and so suggesting that we remove it.

Reply:
It's kind of unfortunate Optical Sight isn't a more common mechanic. I kind of feel like this does warrant a trigger just based on how fast the ground effects resolve, but I'm not totally sure how to word it, and I think `inThenOut` doesn't best convey what should happen. I don't know. It's a tricky question!

Reply:
Yeah, it kind of feels like one of those "this isn't something you can explain in the moment" sort of situations.

Reply:
I'm okay with its removal. The benefit of keeping it and/or just setting get middle/start middle is that the other version of get out is called where as here you would use silence or visually looking at the boss to tell what's upcoming.

Reply:
"Start Middle" seems reasonable.